src/Tooltip.js

```diff
@@ -43,7 +43,11 @@ const Tooltip = React.createClass({
     /**
      * Title text
      */
-    title: React.PropTypes.node
+    title: React.PropTypes.node,
+    /**
+     * Style hash
+     */
+    style: React.PropTypes.object
```

Comment:
I'd actually rather not explicitly document this since it just adds to the clutter, but that's totally just personal preference on my part.